.vscode/settings.json

```diff
@@ -3,15 +3,15 @@
 		"source.fixAll.eslint": "always"
 	},
 	"editor.formatOnSave": true,
-	"eslint.options": {
-		"extensions": [".js", ".vue", ".ts"]
-	},
 	"eslint.validate": [
 		"javascript",
 		"javascriptreact",
+		"json",
+		"jsonc",
 		"typescript",
 		"typescriptreact",
 		"vue"
 	],
-	"javascript.validate.enable": false
+	"javascript.validate.enable": false,
+	"typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Should be removed, this is supposed to be a local setting as it does not work when commited to the repo (security reasons)